compiler/one-cmds/one-init

```diff
@@ -0,0 +1,262 @@
+
+    def write(self, cfg_path):
+        # key: string, value: string
+        sections = {}
+
+        # convert first,
+        for section in self.sections.keys():
+            content = '\n'.join(self.sections[section])
+            sections[section] = content
+
+        with open(cfg_path, 'w') as f:
+            for k in sections.keys():
+                if k != self.default_key:
+                    f.write(k)
+                    f.write('\n')
+                f.write(sections[k])
+                f.write('\n')
+
+    # NOTE: section always start with '[' and end with ']'
+    def set(self, section, key, value):
+        # TODO: Handle exceptions for none of section and key
+        lines = self.sections[section]
+        for index, line in enumerate(lines):
+            # case: comments
+            if line.lstrip().startswith(self.comment_prefixes):
+                continue
+            # case: not strict config
+            if not self.delimiter in line:
+                continue
+
+            words = line.split(self.delimiter, maxsplit=1)
+            if key.strip() == words[0].strip():
+                lines[index] = words[0] + self.delimiter + value
+                return
+
+
+def _get_supported_map():
+    return {
+        'pb': 'tf',
+        'tflite': 'tflite',
+        'bcq': 'bcq',
+        'onnx': 'onnx',
+    }
+
+
+def _supported_model_ext(ext):
+    return ext in _get_supported_map()
+
+
+def _supported_model_type(model_type):
+    return model_type in _get_supported_map()
+
+
+# Return model type by file extension
+def _get_model_type(ext):
+    if not _supported_model_ext(ext):
+        raise NameError('Model extension (' + ext + ') doesn\'t support')
+    return _get_supported_map()[ext]
+
+
+# NOTE: Do not use '_add_default_arg'. '-C' and '-S' options are unnecessary.
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description='command line tool to create a new config file')
+
+    # version
+    parser.add_argument(
+        '-v',
+        '--version',
+        action='store_true',
+        help='show program\'s version number and exit')
+
+    # model path
+    parser.add_argument(
+        '--model_path',
+        required=False,
+        type=str, default='',
+        help='the model path to be used for creating config file')
+
+    return parser
+
+
+def _parse_arg(parser):
+    # NOTE: Trick for parsing args
+    # To handle both `one-init` and `one-init ${model}`,
+    # `one-init ${model}` -> `one-init --model_path ${model}`
+    argv = copy.deepcopy(sys.argv[1:])
+    if len(argv) > 0 and not argv[0].startswith('--'):
+        argv.insert(0, '--model_path')
+    args = parser.parse_args(argv)
+
+    # print version
+    if args.version:
+        _utils._print_version_and_exit(__file__)
+
+    return args
+
+
+def _create_init_cfg(args):
+    template_path = args.template_path
+    dir_path = os.getcwd()
+    cfg_path = os.path.join(dir_path, 'init.cfg')
```

Comment:
cfg file is created in $CWD

Reply:
@YongseopKim , how about using default filename as `one.cfg` or `model.cfg` instead?

Reply:
It doesn't matter. I just started with the name. I'll fix it. (`one.cfg`)